benchmarks/nsinker_spherical_shell/amg.prm

```diff
@@ -0,0 +1,92 @@
+# This is the "Spherical Shell NSinker" benchmark
```

Comment:
Oh, so there is an AMG case. It looks like this is the same as the GMG version, other than the solver section? In this case, I think it would make sense to include one file in the other, rather than copy the whole file. This is also what we do for other benchmarks.